xml/obs_ag_security_concepts.xml

```diff
@@ -181,4 +181,140 @@
    </para>
   </sect2>
  </sect1>
+ <sect1 xml:id="trust_zones">
+  <title>Trust Zones </title>
+  <para>&OBS; components deal with different trust zones. These are
+        seperated via network or virtualization mechanics.
+  </para>
+
+  <sect2 xml:id="public_zone">
+   <title>Public Zones</title>
+   <para>Public zones are areas where any code is running.
+   </para>
+   <sect3 xml:id="public_internet">
+    <title>Public Internet</title>
+    <para>
+      This can be the public Internet if the &obs; instance is a public
+      instance. Requests are only be triggered via http, secured by SSL
+      to the API instance here. Every developer is using this interface
+      to submit changes. However, further components may open
+      connections to the Internet as described below.
+    </para>
+   </sect3>
+   <sect3 xml:id="untrusted_code">
+    <title>Untrusted Code</title>
+    <para>
+      All code which is used to build content is considered
+      to be untrusted code. This includes even the Linux kernel, since
+      users can build and run their own kernels. The security layer here
+      is usually either KVM and XEN virtualization. These instances are
+      running without any network enabled. Build results get copied by 
+      reading a simple blocklist directly from the device. A security 
+      inside of the file system code can therefore not be used to 
+      compromise the worker.
+    </para>
+   </sect3>
+  </sect2>
+  <sect2 xml:id="dmz">
+   <title>Demilitarized Zone (DMZ)</title>
+   <para>The Demilitarized Zone contains service which interacts with
+         the public zone directly. 
+   </para>
+   <sect3 xml:id="frontend">
+    <title>&obs; Frontend</title>
+    <para>
+      The frontend service is the only service which provides an open port.
+      It implements the authorization of user requests. The authentification
+      may happen in an external service like a proxy or ldap server.
+      The frontend reads and writes changes to the database and
+      the source server only.
+    </para>
+   </sect3>
+   <sect3 xml:id="frontend_background">
+    <title>&obs; Frontend Background</title>
+    <para>
+     &obs; frontend background services handle less time critical operations.
+     This includes services which read data from external services like bug
+     trackers, sending notifications or long running jobs.
+    </para>
+   </sect3>
+   <sect3 xml:id="stage_server">
+    <title>Stage Server</title>
+    <para>
+      The stage server is providing the public content of the &obs; backends.
+      The server can be public accessable or just an upload server to
+      a mirror infrastructure.
+    </para>
+   </sect3>
+   <sect3 xml:id="cloud_uploader">
+    <title>Cloud Uploader</title>
+    <para>
+      The cloud uploader handles build results similar to the publisher.
```

Comment:
the publisher is part of the binary_servers .... not sure if I should add this here?

Reply:
I think explaining the publisher is probably not necessary -- I assume the publisher and the other binary services are explained in more detail elsewhere in the document already.

It might be better to just mention what the cloud uploader actually does and remove the reference to the publisher.

Reply:
okay, adapted again.